Gemfile

```diff
@@ -1,6 +1,7 @@
-source 'https://rubygems.org' do
-  gem 'cocoapods', '~> 1'
-  gem 'git', '~> 1'
-  gem 'rake'
-  gem 'jsonlint'
-end
+source 'https://rubygems.org'
+
+gem 'cocoapods', '~> 1'
+gem 'git', '~> 1'
+gem 'rake'
+gem 'jsonlint'
+gem 'rubocop', '~> 1.20'
```

Comment:
My editor gave me some style warnings when touching the `.podspec`. I took the chance to update `Gemfile` to the block-less syntax because we use a single `source`, setup Rubocop and run the `.podspec` through it.